File: radar-auth/src/main/java/org/radarcns/auth/exception/TokenValidationException.java
package org.radarcns.auth.exception;

/**
 * Created by dverbeec on 15/09/2017.
 */
public class TokenValidationException extends RuntimeException {
    public TokenValidationException() {
        super();
    }

    public TokenValidationException(String message) {
        super(message);
    }

    public TokenValidationException(Throwable cause) {
        super(cause);
    }
}

File: radar-auth/src/main/java/org/radarbase/auth/exception/TokenValidationException.kt
package org.radarbase.auth.exception

/**
 * Created by dverbeec on 15/09/2017.
 */
open class TokenValidationException : RuntimeException {
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
    constructor(cause: Throwable) : super(cause)
}
